app/views/friendships/friend_requests.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if @friends_req.empty?%>
+  <h2>No Friend Requests</h2>
+<% else @friends_req %>
+  <h1> Friend Requests </h1>
+  <%= render partial: "friend", collection: @friends_req, :locals => {:greeting => '1'}%>

```

Comment:
Please use new syntax & always a space after `{` & before `}` i.e 
```
locals: { greeting: '1' }
```

For what purpose are we using this variable `greeting`?

Reply:
3 views were using same partial, so to differentiate between them I used this variable